pcl.orogen

```diff
@@ -14,8 +14,26 @@ import_types_from "base"
 import_types_from "pcl/PCLPointCloud2.h"
 import_types_from "PCLPointCloud2WithTimestamp.hpp"
 
+# We have to create a wrapper class for pcl::Vertices, since orogen can't handle that
+# the struct (Vertices) and it's member (vertices) have the same name.
+import_types_from "wrappers/PCLVertices.hpp"
+typekit.opaque_type '/pcl/Vertices', '/wrappers/PCLVertices', :includes => ['boost/shared_ptr.hpp', 'pcl/Vertices.h']
```

Comment:
It will be fixed in PCL 1.8.1 onwards. I added a comment that it can be removed as soon as they are the default version.